index.html

```diff
@@ -71,6 +71,250 @@ <h1 class="headline__title color-dark">
 					</ul>
 				</div>
 			</section>
+			<section class="portfolio" id="experience">
+				<article class="experience grid">
+					<div class="my_portfolio">
+						<header class="nav-port">
+							<img src="img/work/port1.png" class="port-photo">
+						</header>
+						<div class="port-body grid">
+							<h4 class="port-name color-dark">Tonic</h4>
+							<div class="port-description flex-row align-center"></div>
+							<h5 class="port-invent color-info">Canopy</h5>
+							<ul class="job_done flex-row">
+								<li class="items">Back End Dev</li>
+								<li class="items">2015</li>
+							</ul>
+							
+						</div>
+						<p class="description-main">A daily selection of privately personalized reads; no accounts
```

Comment:
Great job. Please make sure your design is the same as Figma design.
Your design

![image](https://github.com/AsciencioAlex/Personal-Portfolio/assets/101523582/0f8f1c79-896e-478a-9b54-8cf1b6bebb86)

Figma design

![image](https://github.com/AsciencioAlex/Personal-Portfolio/assets/101523582/dfd5dc28-9651-4b07-ae7b-3ded19340f8c)